.pipelines/mssql-pipelines.yml

```diff
@@ -49,13 +51,33 @@ jobs:
     inputs:
       dockerVersion: 17.09.0-ce
       releaseType: stable
+  
+  - task: PowerShell@2
+    displayName: 'Generate DB password'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $sourceString = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+        $password = -join ($sourceString.ToCharArray() | Get-Random -Count 20)
+        Write-Host "##vso[task.setvariable variable=dbPassword]$password"
```

Comment:
can we keep consistent password generation between linux steps in various pipelines? DW introduced this: https://github.com/Azure/data-api-builder/blob/40dab94d85b68f538e336e0b732f8cf73fca026e/.pipelines/dwsql-pipelines.yml#L55-L61

Reply:
updated